src/sentry/relay/config/metric_extraction.py

```diff
@@ -354,7 +354,7 @@ def convert_widget_query_to_metric(
     if not widget_query.aggregates:
         return metrics_specs
 
-    if "event.type:error" in widget_query.conditions:
+    if "event.type:error" in widget_query.conditions.split():
```

Comment:
This is not very reliable, since it will fail if a user specified `event.type:"error"`, if I am not wrong. I would suggest to always check anything related to conditions only on the parsed data's AST.